autovalet_lane_detection/src/autovalet_lane_detection/LaneDetector.py

```diff
@@ -78,12 +89,35 @@ def detectLaneRGBD(self, color_img, depth_img):
             publishCloud(ego_line, self.camera.header.frame_id, self.egoLine_pub)
             # maybe publish center line alone too?
 
-            return lane_cloud, ego_line
+            ## Parking directions
+            if  center_line_coordinates.shape[0] % 2 == 0:
+                index = center_line_coordinates.shape[0] / 2 
+            else:
+                index = (center_line_coordinates.shape[0] + 1)/ 2
```

Comment:
These could be refactored into one line code?
`index = int(round((center_line_coordinates.shape[0] + 1) / 2 ))`

Reply:
perfect. done